Resources/Maps/Shuttles/rigger.yml

```diff
@@ -0,0 +1,7588 @@
+      parent: 1
+      type: Transform
+  - uid: 846
+    components:
+    - pos: 6.3446736,10.061763
+      parent: 1
+      type: Transform
+  - uid: 850
+    components:
+    - pos: 6.5151734,9.805839
+      parent: 1
+      type: Transform
+  - uid: 852
+    components:
+    - pos: 6.2807355,9.741858
+      parent: 1
+      type: Transform
+  - uid: 855
+    components:
+    - pos: 6.621737,9.357971
+      parent: 1
+      type: Transform
+  - uid: 856
+    components:
+    - pos: 6.4299245,9.571241
+      parent: 1
+      type: Transform
+  - uid: 857
+    components:
+    - pos: 6.302048,9.272663
+      parent: 1
+      type: Transform
+- proto: DrinkVacuumFlask
+  entities:
+  - uid: 844
+    components:
+    - pos: 6.856175,9.357971
+      parent: 1
+      type: Transform
+  - uid: 851
+    components:
+    - pos: 6.8135514,10.232379
+      parent: 1
+      type: Transform
+  - uid: 853
+    components:
+    - pos: 6.856175,9.613895
+      parent: 1
+      type: Transform
+  - uid: 854
+    components:
+    - pos: 6.8135514,9.912474
+      parent: 1
+      type: Transform
+- proto: EmergencyLight
+  entities:
+  - uid: 272
+    components:
+    - rot: 1.5707963267948966 rad
+      pos: -0.5,0.5
+      parent: 1
+      type: Transform
+  - uid: 276
+    components:
+    - rot: 1.5707963267948966 rad
+      pos: 1.5,-8.5
+      parent: 1
+      type: Transform
+  - uid: 283
+    components:
+    - rot: -1.5707963267948966 rad
+      pos: -0.5,-10.5
+      parent: 1
+      type: Transform
+  - uid: 514
+    components:
+    - rot: -1.5707963267948966 rad
+      pos: -2.5,1.5
+      parent: 1
+      type: Transform
+  - uid: 552
+    components:
+    - rot: 3.141592653589793 rad
+      pos: 2.5,8.5
+      parent: 1
+      type: Transform
+  - uid: 1061
+    components:
+    - pos: -0.5,15.5
+      parent: 1
+      type: Transform
+  - uid: 1062
+    components:
+    - rot: 3.141592653589793 rad
+      pos: -5.5,11.5
+      parent: 1
+      type: Transform
+  - uid: 1063
+    components:
+    - rot: 3.141592653589793 rad
+      pos: -5.5,8.5
+      parent: 1
+      type: Transform
+- proto: ExosuitFabricator
+  entities:
+  - uid: 322
+    components:
+    - pos: -2.5,0.5
+      parent: 1
+      type: Transform
+- proto: ExtinguisherCabinetFilled
+  entities:
+  - uid: 1003
+    components:
+    - rot: 1.5707963267948966 rad
+      pos: -6.5,-2.5
+      parent: 1
+      type: Transform
+  - uid: 1004
+    components:
+    - pos: 3.5,-6.5
+      parent: 1
+      type: Transform
+  - uid: 1006
+    components:
+    - pos: 3.5,7.5
+      parent: 1
+      type: Transform
+  - uid: 1007
+    components:
+    - pos: -1.5,12.5
+      parent: 1
+      type: Transform
+  - uid: 1008
+    components:
+    - rot: 3.141592653589793 rad
+      pos: -4.5,7.5
+      parent: 1
+      type: Transform
+- proto: FaxMachineShip
+  entities:
+  - uid: 527
+    components:
+    - pos: -7.5,11.5
+      parent: 1
+      type: Transform
+- proto: FireAxeCabinetFilled
```

Comment:
Is it bad? Eh, I guess I don't see any problems with removing one of them.

Reply:
Having one in atmos room is sensible I think, but I guess I can delete it, since there are shutters for emergency venting that room. Leaving the one in the bridge.